c/CasADi/CasADi-3.7.0-gfbf-2024a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonBundle'
+
+name = 'CasADi'
+version = '3.7.0'
+
+homepage = 'https://web.casadi.org'
+description = """CasADi is an open-source tool for nonlinear optimization and algorithmic differentiation.
+It facilitates rapid — yet efficient — implementation of different methods for numerical optimal control,
+both in an offline context and for nonlinear model predictive control (NMPC)."""
+
+toolchain = {'name': 'gfbf', 'version': '2024a'}
+dependencies = [('Python', '3.12.3'),
+                ('SciPy-bundle', '2024.05'),
+                ('CMake', '3.29.3')]
+
+exts_list = [
+    ('casadi', version, {
+        'modulename': 'casadi',
```

Comment:
I guess this is only needed if the given extension name is different from the packagename (like different casing, use of dash/underscore, etc) which leads to `import` failure at the pip check test step. I guess in this case, this is not really needed.